libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -616,9 +616,6 @@ describe('Date range picker', function () {
 
     detectChanges();
 
-    expect(control?.errors).toBeFalsy();
-    expect(calculatorIdControl?.errors).toBeFalsy();
```

Comment:
Shouldn't these still be true?

Reply:
no bc with the change and datepickers always being required, changing calculators does mark the date range picker as required. so it will have errors in this state

Reply:
Ah, because the test is setting the calculator ID to `SpecificRange` with empty dates. Cool.